### PR TITLE
Autoplay setting

### PR DESCRIPTION
**Pull request by originalwebgurl**
Added autoplay persistence.

- If autoplay is in url search params it will override the board setting value (and/or localstorage value)
- can persist autoplay setting in localstorage
- the autoplay setting defaults to true

### CODE REVIEW

File: src/app/store.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const store = configureStore({
     game: rootReducer,
     settings: settingsReducer
   },
+  // preloadedState: rehydrateStore,
```
Comment (bvanvugt) [inline src/app/store.js:12]: Residual debug code?

**Review by bvanvugt — APPROVED** [review]

File: src/app/storage.js [hunk 2]
```diff
@@ -24,3 +24,31 @@ export function storageAvailable(type) {
...
+  }
+}
+
+export function rehydrateLocalSettings() {
+  let checkAutoplay = getLocalSetting("autoplay");
+
+  if (typeof checkAutoplay === "undefined") {
+    checkAutoplay = true; // default to true
+  } else {
+    checkAutoplay = checkAutoplay === "true";
```
Comment (bvanvugt) [inline src/app/storage.js:46]: I might be tracing it wrong, but I think it's being stored as a bool... ?

File: src/app/store.js (hunk 1 above)
Comment (originalwebgurl) [reply src/app/store.js:12]: Deleting...

File: src/app/storage.js (hunk 2 above)
Comment (originalwebgurl) [reply src/app/storage.js:46]: So localStorage stores everything as a string. We need to convert all non-string values.

**Pull request merged**